src/ViewInstance.js

```diff
@@ -657,6 +664,19 @@ class ViewInstance extends React.Component {
     const canCreateOrder = !checkIfUserInCentralTenant(stripes) && stripes.hasInterface('orders') && stripes.hasPerm('ui-inventory.instance.createOrder');
     const canReorder = stripes.hasPerm('ui-requests.reorderQueue');
     const canExportMarc = stripes.hasPerm('ui-data-export.app.enabled');
+
+    const hasSetForDeletionPermission = stripes.hasPerm(setForDeletionAndSuppressPerm);
+    const canCentralTenantSetForDeletion = hasSetForDeletionPermission && isShared;
+    const canMemberTenantSetForDeletion = (isShared && this.hasCentralTenantPerm(setForDeletionAndSuppressPerm)) || (!isShared && hasSetForDeletionPermission);
+    const canSetForDeletion = hasSetForDeletionPermission || canCentralTenantSetForDeletion || canMemberTenantSetForDeletion;
+
+    const isRecordSuppressed = instance.discoverySuppress && instance?.staffSuppress;
```

Comment:
Why do you have a check for `instance` for `staffSuppress` but do not have for `discoverySuppress`?
```suggestion
    const isRecordSuppressed = instance?.discoverySuppress && instance?.staffSuppress;
```
or
```suggestion
    const isRecordSuppressed = instance.discoverySuppress && instance.staffSuppress;
```